module/Finna/src/Finna/Controller/MyResearchController.php

```diff
@@ -655,16 +655,9 @@ public function changeProfileAddressAction()
                     $this->flashMessenger()->addErrorMessage($e->getMessage());
                 }
             } else {
-                if (!isset($updateConfig['emailAddress'])) {
-                    throw new \Exception(
-                        'Missing emailAddress in ILS updateAddress settings'
-                    );
-                }
-                $recipient = $updateConfig['emailAddress'];
-
-                $this->sendChangeRequestEmail(
-                    $patron, $profile, $data, $fields, $recipient,
-                    'Yhteystietojen muutospyyntö', 'change-address'
+                $this->saveChangeRequestFeedback(
+                    $patron, $profile, $data, $fields,
+                    'UpdatePersonalInformation', 'change-address'
```

Comment:
FeedbackForms.yaml.sampleen voisi lisätä tämän "lomakkeen", niin ei tule vahingossa yliajettua asetustiedostossa, esim:

# Reserved for Axiell update messaging settings form                                                                                                                                        UpdateMessagingSettings:

Reply:
Lisäilen kommentin ja muokkaan taulun alkamaan finna_ jotta se on selkeämpi?

Reply:
lisätty kommenttiin huomio

Reply:
Lisäsin vielä palautusarvon ja virheilmoituksen jos asetus puuttuu viestiä lähetettäessä.